applications/GeoMechanicsApplication/custom_elements/transient_thermal_element.h

```diff
@@ -74,6 +74,8 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) TransientThermalElement : public Ele
         if (GetGeometry().LocalSpaceDimension() == 1) {
             GetGeometry().DeterminantOfJacobian(det_J_container, this->GetIntegrationMethod());
             dN_dX_container = GetGeometry().ShapeFunctionsLocalGradients(this->GetIntegrationMethod());
+            std::transform(dN_dX_container.begin(), dN_dX_container.end(), det_J_container.begin(),
```

Comment:
Is there a way to test this bug without having point fluxes? If not, I'm okay to only have the tests in that PR.

Reply:
@rfaasse Thank you for the review.
I used to test it without flux, but it was converging to correct steady state. It seems the cases with Dirichlet conditions were not affacted much. This problem appeared when added test cases with flux.
There is maybe possible to follow the unsteady path. But as we don't have an accurate unsteady development, a such test may or may not give an accurate indication.